server/eb/app.py

```diff
@@ -185,7 +185,7 @@ def get_csp_hashes(app, app_config):
 
 if __name__ == "__main__":
     try:
-        application.run(debug=debug, threaded=not debug, use_debugger=False)
+        application.run(host=app_config.server_config.app__host, debug=debug, threaded=not debug, use_debugger=False)
```

Comment:
is this dependent on changes in a different pr?

Reply:
No this is needed for docker.    I need to pass in "0.0.0.0" for the host so that it can server outside requests.

Reply:
There is a change in single-cell-infra that passes in host as 0.0.0.0